app/templates/components/public/session-list.hbs

```diff
@@ -0,0 +1,8 @@
+{{#each-in sessions as |id trackSession|}}
```

Comment:
There is no need to handle this as output of both filter and unfiltered data is always going to be an object and not an array.
The format:
`{ track1: Array, track2: Array ...}`